docs/topic-guides/workflows.rst

```diff
@@ -0,0 +1,100 @@
+.. _workflows:
+
+Renku Workflows
+===============
+
+Together with :ref:`provenance of results <provenance>`, Renku workflows is
+a key feature of Renku to make code and processing pipelines reusable.
+When tracking executions with Renku, each execution creates a workflow step.
+These steps can be joined together into bigger workflows, making it easy to
+run them with varying inputs and on different execution backends.
+
+There are many uses for this. You could define and test a pipeline locally
+and then execute it on an HPC cluster on a big dataset once you're happy.
+Or you could run your workflow offer a range of different parameters to see
+how the results differ. Or you create a workflow that runs daily with a
+CRON job.
+
+All executions of workflows are also tracked in the provenance, without you
+having to re-type and memorize the commands used in each step.
+
+Working with Workflows
+----------------------
+
+Tracking execution of a command with :meth:`renku run <renku.ui.cli.run>`:
+
+.. code-block:: console
+
+    $ renku run --name run-analysis -- python run_analysis.py -i inputs -o outputs
+
+creates a workflow step called `run-analysis`. You can inspect the workflow
```

Comment:
do you mean to be using single ticks here (italics) instead of double ticks (code styling) ?